trunk/source/CACSL2BoogieTranslator/src/de/uni_freiburg/informatik/ultimate/cdt/translation/implementation/base/chandler/MemoryArea.java

```diff
@@ -0,0 +1,12 @@
+package de.uni_freiburg.informatik.ultimate.cdt.translation.implementation.base.chandler;
+
+public enum MemoryArea {
+	STACK, HEAP,;
```

Comment:
Remove the extra comma to avoid further (unnamed) enum constant.

```suggestion
	STACK, HEAP;
```

Reply:
[4c4ef60](https://github.com/ultimate-pa/ultimate/pull/756/commits/4c4ef60303828185ffc4cc3fd1cfce1ae892b79b)